### PR TITLE
UX improvement on selecting HTTPS versus HTTP mode

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

_What behavior does this PR change, and why?_

Pull logic into the start script, plus minor tweaks

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

**Review by mikebin — APPROVED** [review]
LGTM

File: scripts/start.sh [hunk 1]
```diff
@@ -14,6 +14,20 @@ export REPOSITORY=${REPOSITORY:-confluentinc}
...
 export CONNECTOR_VERSION=${CONNECTOR_VERSION:-$CONFLUENT}
 
+# Control Center and ksqlDB server must both be HTTP or both be HTTPS; mixed modes are not supported
+# C3_KSQLDB_HTTPS=false: set Control Center and ksqlDB server to use HTTP (default)
+# C3_KSQLDB_HTTPS=true : set Control Center and ksqlDB server to use HTTPS
+C3_KSQLDB_HTTPS=${C3_KSQLDB_HTTPS:-false}
+if [[ "$C3_KSQLDB_HTTPS" == "false" ]]; then
+  export CONTROL_CENTER_KSQL_WIKIPEDIA_URL="http://ksqldb-server:8088"
+  export CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL="http://localhost:8088"
+  C3URL=http://localhost:9021
```
Comment (javabrett) [inline scripts/start.sh:24]: All other `URL` variables are preceded by an underscore, consider `C3_URL` here.

File: scripts/start.sh [hunk 2]
```diff
@@ -14,6 +14,20 @@ export REPOSITORY=${REPOSITORY:-confluentinc}
 # and expects user to build and provide a local file confluentinc-kafka-connect-replicator-${CONNECTOR_VERSION}.zip
 export CONNECTOR_VERSION=${CONNECTOR_VERSION:-$CONFLUENT}
 
+# Control Center and ksqlDB server must both be HTTP or both be HTTPS; mixed modes are not supported
+# C3_KSQLDB_HTTPS=false: set Control Center and ksqlDB server to use HTTP (default)
+# C3_KSQLDB_HTTPS=true : set Control Center and ksqlDB server to use HTTPS
+C3_KSQLDB_HTTPS=${C3_KSQLDB_HTTPS:-false}
+if [[ "$C3_KSQLDB_HTTPS" == "false" ]]; then
+  export CONTROL_CENTER_KSQL_WIKIPEDIA_URL="http://ksqldb-server:8088"
```
Comment (javabrett) [inline scripts/start.sh:22]: These two would prevent me from setting these separately in my own environment, should I wish to do that for some reason.  I'm not that fussed, but should we allow pre-set via `:-` here and following line?

**Review by javabrett — APPROVED** [review]
Nice - simplifies enabling HTTPS without needing to know the proper settings for ksqlDB server.

File: scripts/start.sh (hunk 1 above)
Comment (ybyzek) [reply scripts/start.sh]: Ack and fixed

File: scripts/start.sh (hunk 2 above)
Comment (javabrett) [reply scripts/start.sh:22]: I guess the problem here is that this variable has to be exported (does it?) so wouldn't get reset ... ignore this I think.

Comment (ybyzek) [reply scripts/start.sh]: Ack and resolved

**Pull request merged**